### ISSUE TITLE
Kucoin: Bad API normalization for BSV?

### ISSUE THREAD

**Issue by fbadiola** [ccxt/ccxt#6538]
- OS: Linux
- Programming Language version: JavaScript
- CCXT version: 1.23.9
- Exchange: Kucoin
- Method: fetchMarkets

Hello,

I just seeing Kucoin client and I think that has got a bad API normalization for BSV. Kucoin API uses `BCHSV` as a ID for the currency `BSV`:

If we see the [currencies list](https://openapi-v2.kucoin.com/api/v1/currencies) we can found this:
```json
{
  "withdrawalMinFee": "0.0005",
  "isDebitEnabled": true,
  "precision": 8,
  "name": "BSV",
  "fullName": "BCHSV",
  "currency": "BCHSV",
  "withdrawalMinSize": "0.001",
  "isWithdrawEnabled": true,
  "isDepositEnabled": true,
  "isMarginEnabled": true
}
```

Also we can see this endpoints:

https://openapi-v2.kucoin.com/api/v2/market/orderbook/level2?symbol=BCHSV-USDT

https://openapi-v2.kucoin.com/api/v2/market/orderbook/level2?symbol=BSV-USDT (not supported)

So currency field (BCHSV) is kucoin's identifier and its name is BSV.

We can reproduce it with this code:

```javascript
const ccxt = require('ccxt');

const exchange = new ccxt.kucoin();

async function main() {
  const markets = await exchange.fetchMarkets();

  console.log(markets.find(x => x.base === 'BSV').baseId);
}

main();
```

Is this correct?

Thanks for your attention (and work) !!!

**Comment by kroitor** [comment]
@fbadiola if you open this example:

- https://github.com/kroitor/ccxt.pro/blob/master/examples/js/cli.js

```
node examples/js/cli kucoin currencies --table
```

It does output the table of currencies

```
mbp:ccxt igorkroitor$ node examples/js/cli kucoin currencies --table
        id |                                   name |       code | precision
----------------------------------------------------------------------------
...
    BCHABC |                                 BCHABC |        BCH |         8
     BCHSV |                                  BCHSV |        BSV |         8
...
240 objects
```

> Kucoin API uses BCHSV as a ID for the currency BSV:

That's right and this is explained in the Manual: 

- https://github.com/ccxt/ccxt/wiki/Manual#symbols-and-market-ids
- https://github.com/ccxt/ccxt/wiki/Manual#naming-consistency

> So currency field (BCHSV) is kucoin's identifier and its name is BSV.

Exactly. This is how it should be, since all the world uses the `BSV` now to designate that currency, and some exchanges may still be using BCHSV, which is ok.

Let me know if the above does not answer your question or if we are missing something.

**Comment by fbadiola** [comment]
Maybe I explained me bad.

If you call `kucoin.fetchMarkets()` , ccxt returns as baseId: 'BSV' when 'BCHSV' is the correct.

**Comment by kroitor** [comment]
@fbadiola you're right, thx for the clarification! I've uploaded a fix for it and it should be fine in 1.23.10 (the new build will become available in 15 minutes). Let me know if you have any difficulties with the new version. It will be great if you report back whether the issue is resolved or not.

**Comment by fbadiola** [comment]
Its corrected in these version. Thanks. 💕💕
